<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HID Device</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>537</width>
     <height>214</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="mLoCB">
      <item>
       <widget class="QComboBox" name="mListDevs"/>
      </item>
      <item>
       <widget class="QPushButton" name="mBtnUpdate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>О&amp;бновить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="mLoConDis">
      <item>
       <widget class="QPushButton" name="mBtnConnect">
        <property name="text">
         <string>&amp;Подключиться</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mBtnDisconnect">
        <property name="text">
         <string>&amp;Отключиться</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mBtnExit">
        <property name="text">
         <string>&amp;Выход</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="mLoCheckBox">
      <item>
       <widget class="QCheckBox" name="mChkLed">
        <property name="text">
         <string>&amp;Led on Board</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mChkKey">
        <property name="text">
         <string>BT &amp;Key</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mChkReset">
        <property name="text">
         <string>BT &amp;Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="mLoLeds">
      <item>
       <widget class="QGraphicsView" name="mLedBoard">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="mBoot1Board">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="mLed1BT">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="mLed2BT">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="mKeyBT">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="mRstBT">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="backgroundBrush">
         <brush brushstyle="NoBrush">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </property>
        <property name="foregroundBrush">
         <brush brushstyle="NoBrush">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </property>
        <property name="interactive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
